### PR TITLE
Feat/add numeric value ranges

### PR DESCRIPTION
**Pull request by ojwanganto**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Display value ranges for numeric test results

## Screenshots
image

### CODE REVIEW

File: src/results/result-form-field.component.tsx [hunk 1]
```diff
@@ -23,6 +24,28 @@ const ResultFormField: React.FC<ResultFormFieldProps> = ({
...
+      minVal = Math.min(
+        concept?.lowAbsolute,
+        concept?.lowCritical,
+        concept?.lowNormal
+      );
+      rangeString = ` (${minVal ?? 0} - ${maxVal ?? 0} ${
+        concept?.units ?? ""
+      })`;
+    }
+    return rangeString;
```
Comment (jabahum) [inline src/results/result-form-field.component.tsx:46]: can we make the ranges or the Test bold atleast

Comment (ojwanganto) [reply src/results/result-form-field.component.tsx:46]: How is that done? inline styling?

File: src/results/result-form.resource.ts [hunk 2]
```diff
@@ -19,6 +19,12 @@ export interface ConceptResponse {
   attributes: any[];
   links: Link18[];
   resourceVersion: string;
+  hiNormal: number;
+  hiAbsolute: number;
+  hiCritical: number;
+  lowNormal: number;
+  lowAbsolute: number;
+  lowCritical: number;
```
Comment (pirupius) [inline src/results/result-form.resource.ts:27]: Are these generic to all concept responses? if not we should make them ternary 

```suggestion
  hiNormal?: number;
  hiAbsolute?: number;
  hiCritical?: number;
  lowNormal?: number;
  lowAbsolute?: number;
  lowCritical?: number;
```

File: src/results/result-form.resource.ts [hunk 3]
```diff
@@ -159,6 +165,13 @@ export interface ConceptReference {
   attributes: any[];
   links: Link15[];
   resourceVersion: string;
+  hiNormal: number;
+  hiAbsolute: number;
+  hiCritical: number;
+  lowNormal: number;
+  lowAbsolute: number;
+  lowCritical: number;
```
Comment (pirupius) [inline src/results/result-form.resource.ts:173]: same comment applies here about adding the ternary

**Review by jabahum — APPROVED** [review]

**Comment by ojwanganto** [comment]
Review comments addressed. Please check. @jabahum @pirupius @donaldkibet

**Review by pirupius — APPROVED** [review]

**Pull request merged**